playbooks/ocp-install-preparation.yml

```diff
@@ -1,64 +1,48 @@
 ---
+- name: OCP | Install OpenShift Ansible packages on installer
+  hosts: localhost
+  vars_files:
+    - '../defaults/main.yml'
+  become: yes
+  tasks:
+    - name: Install OpenShift Ansible packages
+      package:
+        name: "openshift-ansible*"
+        state: latest
+      tags:
+        installer-prep
+      when: ansible_facts['distribution'] == "RedHat"
+
 - name: OCP | Host Registration
   hosts: all
+  become: yes
+  vars_files:
+    - '../defaults/main.yml'
   tasks:
     - name: Disable all repos
-      command: "subscription-manager repos --disable='*'"
+      rhsm_repository:
```

Comment:
@pekramp get rid of this because it wont be idempotent with it, it was a hack because we didn't have a module. see line 28 for better way.

Reply:
After trying to get it to work this morning, Eric pointed out why it wasn't working:
"purge boolean added in 2.8"
and since OpenShift 3.11 only supports 2.6 it doesn't work.

Reply:
Well that is annoying.